src/test/java/Tests/object/BCommentTable.java

```diff
@@ -62,6 +74,22 @@ public boolean isCommentInactive(String state) {
         }
     }
 
-}
-
+    @Step("Verify presence of the category in comment categories ")
+    public boolean verifyCategories(String expected) throws NoSuchElementException {
+        try {
+            List<WebElement> categories = this.driver.findElements(By.xpath("//*[@class='categorycolumn']"));
+            for (WebElement row : categories) {
+                int i = 0;
+                row = categories.get(i);
+                String actual = row.getText();
+                System.out.println(row.getText());
+                if (actual.contains(expected))
+                    return true;
+            }
+        } catch (NoSuchElementException e) {
+            System.out.println(e);
```

Comment:
It's antipattern just to print exception. You have to handle it (make a `return`) or throw a new one. If you don't use either of options, just remove `try - catch` block.

Reply:
fixed

I don't need try-catch block here (I think).